capstone-frontend/src/components/BookDescriptionOverlay.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Popover, OverlayTrigger } from 'react-bootstrap';
+
+const popoutDescription = (book) => {
+  let description = book.description || "";
+
+  if (description.length > 500) {
+    description = description.substring(0, 500).concat('...');
+  }
+
+  return (
+    <Popover>
+      <Popover.Title as='h3'>
+        <Link to={`/bookpage/${book.id}`} className='center-horizontal'
```

Comment:
nit: This would probably be more readable with each prop on its own line. Follow this pattern when they don't all fit on the same line.

Reply:
True, this does make it much more readable and consistent!